net-libs/libsrtp/libsrtp-1.5.4.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Open-source implementation of the Secure Real-time Transport Protocol (SRTP)"
+HOMEPAGE="https://github.com/cisco/libsrtp"
+SRC_URI="https://github.com/cisco/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/1"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 -sparc ~x86 ~x86-fbsd ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="aesicm console debug doc libressl openssl static-libs syslog test"
+
+REQUIRED_USE="libressl? ( openssl )"
```

Comment:
media-video/ffmpeg has a similar problem to what I'm trying to avoid in this ebuild. It should actually have a REQUIRED_USE="libressl? ( openssl )", because if you don't enable both, the libressl USE flag does nothing.

This ebuild does not unconditionally depend on OpenSSL, it has an optional dependency on OpenSSL to accelerate their crypto functions.

Alternatively, I can modify the line $(use_enable openssl) to pass '--enable-openssl' to configure regardless of whether we are using OpenSSL or LibreSSL.